repositories/repositories.bzl

```diff
@@ -1,17 +1,15 @@
 """Handles imports of external/third-party repositories.
 """
 
-load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 load(
     "@com_github_mvukov_rules_ros2//repositories:ros2_repositories_impl.bzl",
     "ros2_repositories_impl",
 )
 
-def ros2_repositories():
-    """Imports external/third-party repositories.
-    """
+def rules_ros2_workspace_deps():
```

Comment:
Well, let's be consistent. I'd name this e.g. ros2_workspace_repositories(). IMO, this is still a bit ambiguous compared to ros_repositories, but the docs should do the explanation. IMHO, the suffix should be _repositories, not _deps. We use _deps for configuration of repositories.

Another alternative is to keep this as ros2_repositories and rename the other one as ros2_package_repositories() (which could be more specific).

WDYT?

Reply:
Don't care too much, the best move would be to migrate to Bzlmod and then just drop Workspace altogether :D